src/locales/en/common.json

```diff
@@ -121,12 +124,12 @@
   "Delegate": "Delegate",
   "Delegate #{{rank}}": "Delegate #{{rank}}",
   "Delegate profile": "Delegate profile",
+  "Delegate registration Summary": "Delegate registration Summary",
   "Delegate registration succeeded": "Delegate registration succeeded",
-  "Delegate registration summary": "Delegate registration summary",
   "Delegate weight": "Delegate weight",
   "Delegates": "Delegates",
   "Delegates overview": "Delegates overview",
-  "Delegation status": "Delegation status",
+  "Delegation Status": "Delegation Status",
```

Comment:
any particular reason why we choose `Delegation` instead of `Delegate`?

Reply:
no. I just changed the S in status. Didn't want to touch the text in case some test depends on it.

Reply:
the uppercase change i understood, but i dint get choose in the past to use `Delegation` terminology. no need to change, if it was intended to be `Delegation` then all good.

Reply:
Changed.